### PR TITLE
Update back-up-docker-installed-rancher.md

### PR DESCRIPTION
**Pull request by asloan7**
"Then" to "Optionally"

### CODE REVIEW

File: docs/how-to-guides/new-user-guides/backup-restore-and-disaster-recovery/back-up-docker-installed-rancher.md [hunk 1]
```diff
@@ -58,7 +58,7 @@ This procedure creates a backup that you can restore if Rancher encounters a dis
 
 1. Enter the `ls` command to confirm that the backup tarball was created. It will have a name similar to `rancher-data-backup-<RANCHER_VERSION>-<DATE>.tar.gz`.
 
-1. Move your backup tarball to a safe location external to your Rancher Server. Then delete the `rancher-data-<DATE>` container from your Rancher Server.
+1. Move your backup tarball to a safe location external to your Rancher Server. Optionally delete the `rancher-data-<DATE>` container from your Rancher Server.
```
Comment (btat) [inline docs/how-to-guides/new-user-guides/backup-restore-and-disaster-recovery/back-up-docker-installed-rancher.md:61]:
```suggestion
1. Move your backup tarball to a safe location external to your Rancher Server. Optionally, delete the `rancher-data-<DATE>` container from your Rancher Server.
```

**Review by btat — APPROVED** [review]

**Comment by asloan7** [comment]
Sorry, I figured out what it should really be now, this can be cancelled I will resubmit

**Comment by btat** [comment]
> Sorry, I figured out what it should really be now, this can be cancelled I will resubmit

Will do, thanks for the update.

**Comment by asloan7** [comment]
New PR is here https://github.com/rancher/rancher-docs/pull/301

**Pull request closed without merging**